chart/chaoskube/README.md

```diff
@@ -0,0 +1,14 @@
+# chaoskube Helm Chart
+chaoskube periodically kills random pods in your Kubernetes cluster.
+
+## Installation
+
+### Helm 3: install the chaoskube Helm Chart
+```
+helm repo add chaoskube https://chaoskube.github.io/chaoskube-kubernetes
```

Comment:
Basically you can publish a chart on any webserver, which can serve static files. There are some opensource projects which use github pages to host thier helm repos. There is also a statement about this in the [Helm documentation](https://helm.sh/docs/topics/chart_repository/#github-pages-example).

Ingress-nginx is a popular example, see <https://kubernetes.github.io/ingress-nginx/index.yaml>. They use github pages to serve the index.yaml and normal github releases to serve the chart. I'm open to any solution, depends a bit on you depending what you like/dislike.

Reply:
There is also a [chart-releaser github action](https://github.com/helm/chart-releaser-action), which looks very simple from my point of view. Looks like ingress-nginx is using this action too [see here](https://github.com/kubernetes/ingress-nginx/blob/master/.github/workflows/helm.yaml). I can also add such a github action if you're ok with that!

Reply:
Github Actions sounds good. https://github.com/helm/chart-releaser-action looks promising. I'll be happy if you give it a try. I can also check in case you get any permission problems etc.

Reply:
chart-releaser works, did a release in my fork you can test it with the following commands:

```plaintext
$> helm repo add chaoskube https://ghouscht.github.io/chaoskube/
"chaoskube" has been added to your repositories

$> helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "chaoskube" chart repository
Update Complete. ⎈Happy Helming!⎈

$> helm template chaoskube chaoskube/chaoskube
---
# Source: chaoskube/templates/serviceaccount.yaml
apiVersion: v1
kind: ServiceAccount
```

Reply:
Does the branch merely need to exist or do we want to merge-in https://github.com/ghouscht/chaoskube/tree/gh-pages?